src/renderer/component/video/internal/play-button.jsx

```diff
@@ -1,47 +1,42 @@
+// @flow
 import React from 'react';
-import Link from 'component/link';
+import Button from 'component/link';
 
-class VideoPlayButton extends React.PureComponent {
-  componentDidMount() {
```

Comment:
Play/pause from the spacebar seems to work fine without this. Maybe it's need for other OS's?

Reply:
yeah it should be fine without this... ill check linux

Reply:
yeah it should be fine without this, I will check linux

Reply:
ok i figured our why it was there.

When you load the page, don't click anything, and press the space bar, this starts the video / initiates the download. When you take this out, it no longer works -- at least on linux.

That said, I think its better to take out this functionality, since we are going to implement autoplay soon anyway.